pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -1,28 +1,46 @@
 package kubernetes
 
 import (
+	"bytes"
 	"context"
+	"fmt"
 	"io"
 	"os"
+	"strings"
+	"time"
 
 	"github.com/woodpecker-ci/woodpecker/pipeline/backend/types"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+
+	// To authenticate to GCP K8s clusters
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
```

Comment:
Add noContext var for the previously suggested fix
```suggestion

var noContext = context.Background()

```